### PR TITLE
Performance stats

### PR DESCRIPTION
**Pull request by neilcook**

### CODE REVIEW

File: wforce.cc [hunk 1]
```diff
@@ -695,6 +695,8 @@ char* my_generator(const char* text, int state)
       "showNamedReportSinks()",
       "addNamedReportSink(",
       "setNamedReportSinks(",
+      "showPerfStats()",
```
Comment (cmouse) [inline wforce.cc]: in wrong commit

File: docs/manpages/wforce.1.md [hunk 2]
```diff
@@ -150,6 +150,27 @@ started with the -c option.
 		 custom1                        true
 		 custom2                        false
 
+* showPerfStats() - Returns information about performance
```
Comment (cmouse) [inline docs/manpages/wforce.1.md]: showStringStatsDB missing

Comment (neilcook) [inline docs/manpages/wforce.1.md]: Not sure what you mean - showStringStatsDB is documented in wforce.1.md already

File: wforce.cc (hunk 1 above)
Comment (neilcook) [inline wforce.cc]: yep will split

File: docs/manpages/wforce.1.md (hunk 2 above)
Comment (cmouse) [inline docs/manpages/wforce.1.md]: sorry, my bad.

**Pull request merged**